Augurpy.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
```

Comment:
<p>So what does the plot show us? Maybe add one sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>